src/instance.js

```diff
@@ -82,6 +82,11 @@ Instance.getTypeType_ = function(type) {
  * @param {object} [options] See {@link Bigtable#createInstance}.
  * @param {object} [options.gaxOptions]  Request configuration options, outlined
  *     here: https://googleapis.github.io/gax-nodejs/global.html#CallOptions.
+ * @param {function} callback The callback function.
```

Comment:
These are the response parameters: https://github.com/muraliQlogic/nodejs-bigtable/blob/9049097a93e294576dd3cf0fed639678e1fa7c9a/src/index.js#L447-L453